docs/guides/blocks/sync-tables/two-way.md

```diff
@@ -0,0 +1,300 @@
+
+```ts
+const ShirtSchema = coda.makeObjectSchema({
+  properties: {
+    size: {
+      type: coda.ValueType.String,
+      autocomplete: ["S", "M", "L", "XL"],
+    },
+    color: {
+      type: coda.ValueType.String,
+      autocomplete: async function (context) {
+        let size = context.propertyValues.size;
+        let availableColors = await getColorsForSize(context, size);
+        return availableColors;
+      },
+    },
+    pattern: {
+      type: coda.ValueType.String,
+      autocomplete: async function (context) {
+        let search = context.search;
+        let patterns = await searchPatterns(context, search);
+        return patterns;
+      },
+    },
+  },
+  // ...
+});
+```
+
+For sync tables with dynamic schemas, you aren't allowed to define the search function directly on the property itself. Instead use the special value `AutocompleteType.Dynamic`, which tells Coda to call the sync table's `autocomplete` function. This is a single function that handles the auto-complete for all dynamic properties. It's defined directly on a dynamic sync table, or within the `dynamicOptions` of a regular sync table. The function can determine which property to provide autocomplete for by inspecting `context.propertyName`.
+
+=== "Dynamic sync table"
+
+    ```ts
+    pack.addDynamicSyncTable({
+      // ...
+      getSchema: async function (context) {
+        let attributes = await getCustomAttributes(context);
+        for (let attr of attriutes) {
+          properties[attr] = {
+            // ...
+            autocomplete: coda.AutocompleteType.Dynamic,
+          }
+        }
+        // ...
+      },
+      autocomplete: async function (context) {
+        let attr = context.propertyName;
+        let options = await getCustomAttributeOptions(context, attr);
+        return options;
+      },
+    });
+    ```
+
+=== "Regular sync table"
+
+    ```ts
+    pack.addSyncTable({
+      // ...
+      dynamicOptions: {
+        getSchema: async function (context) {
+          // ...
+          let attributes = await getCustomAttributes(context);
+          for (let attr of attriutes) {
+            properties[attr] = {
+              // ...
+              autocomplete: coda.AutocompleteType.Dynamic,
+            }
+          }
+          // ...
+        },
+        autocomplete: async function (context) {
+          let attr = context.propertyName;
+          let options = await getCustomAttributeOptions(context, attr);
+          return options;
+        },
+      },
+    });
+    ```
+
+
+## Handling errors
+
+When a row fails to update Coda will show an error message attached to that row. The pending edits will be retained, so that they can adjust them and try again.
```

Comment:
Nit, "so that they" seems to be a referencing a user who is not mentioned, maybe change to "so that the user can adjust them".

Reply:
Good suggestion, done.